app/views/stories/_main_stories_feed.html.erb

```diff
@@ -1,3 +1,12 @@
+<% if should_show_latest_spam_suppression? %>
```

Comment:
What do you think about passing `stories` explicitly? No need to disable rubocop as a bonus.

Reply:
:+1: